### PR TITLE
Final UI Updates

### PR DESCRIPTION
**Pull request by pryanalewis**
**Things to Check**
-If you could briefly look through all the pages to see if I missed anything.
-I updated the chat page so that if a users has no classes, the send message bar is hidden
-Implemented chat titles that display the course code above the chat window
-Slightly increased font-size and changes some text from black to white

-Updates-
- added text to the chat div for when there are no classes in a users list
- added additional comments to the chat controller

### CODE REVIEW

**Review by TDinaso559 — CHANGES_REQUESTED** [review]
Everything is working like you said. I like all the changes except for the part when you don't have any classes, then you don't see anything in the chat. Instead of it just being a big white box, can you make it say something like "go add a class to start chatting"

**Comment by pryanalewis** [comment]
@TDinaso559 Yea I wasn't sure if I should add a message since it already says to the left in the list of classes div to add a class. Wasn't sure if it would be redundant. I can easily add it though.

**Comment by TDinaso559** [comment]
I’d say add something!

**Review by TDinaso559 — APPROVED** [review]
I like it! Good work!!

File: modules/chat/client/controllers/chat.client.controller.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ angular.module('chat').controller('ChatController', ['$scope', '$location', 'Aut
     $scope.classes = [];
     $scope.currentlySelectedClassID;
     $scope.isDefaultSet = false;
+    $scope.currentlySelectedCourseCode;
+
```
Comment (Bakushin10) [inline modules/chat/client/controllers/chat.client.controller.js]: Could you provide the comment for what $scope.currentlySelectedCourseCode is for ? We also have  $scope.currentlySelectedClassID which is used to switch the chat room. these two names are very similar and I want to make sure what each variable is for.

**Review by Bakushin10 — CHANGES_REQUESTED** [review]

File: modules/chat/client/controllers/chat.client.controller.js (hunk 1 above)
Comment (pryanalewis) [reply modules/chat/client/controllers/chat.client.controller.js]: I only created the currentlySelectedCourseCode. The currentlySelectedClassID was already there and it's the generated id from the database to determine the currently selected class for the chat. But I needed to display the actual course code so I created the other variable to display the current course code.

**Review by Bakushin10 — APPROVED** [review]

**Review by yotoo2920 — APPROVED** [review]
I think you have covered all the inputs from the professor. It is looking good!!!!

**Pull request merged**